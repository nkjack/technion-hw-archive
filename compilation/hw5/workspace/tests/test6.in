
void foo(int a)
@pre(a > 0 and a < 15)
{
	print("OK\n");
}

void maxTestsAssignVarToVar2() {
	int x = 3;
	int y = 5;
	int temp;
	temp = x;
	x = y;
	y = temp;
	printi(x);
	print("\n");
	printi(y);
}

void main() {
	foo(20);
     maxTestsAssignVarToVar2();
     print("\nI hate this course\n");
}
