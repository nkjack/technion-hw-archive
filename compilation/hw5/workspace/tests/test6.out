Loaded: ./exceptions.s
Precondition hasn't been satisfied for function foo
